demos/zoom-variations.html

```diff
@@ -27,9 +27,14 @@
 			let variations = [
 				['X only', { x: true, y: false }],
 				['Y only', { x: false, y: true }],
+				['X only + min distance', { x: true, y: false, dist: 50 }],
+				['Y only + min distance', { x: false, y: true, dist: 50 }],
 				['X or Y (adaptive)', { x: true, y: true, uni: Infinity }],
+				['X or Y (adaptive) + min distance', { x: true, y: true, uni: Infinity, dist: 50 }],
 				['X or Y (omni)', { x: true, y: true }],
+				['X or Y (omni) + min distance', { x: true, y: true, dist: 50 }],
 				['X or Y (adaptive + omni)', { x: true, y: true, uni: 50 }],
+				['X or Y (adaptive + omni) + min distance', { x: true, y: true, uni: 50, dist: 50 }],
```

Comment:
i would place all the `dist` demos at the end (maybe even after some `<h2>` for clarity, otherwise it might feel like a bug that the selection doesnt start immediately.

also, while dist: 50 is obvious, it's pretty crazy. i think the majority case would be to prevent accidental drags, so something like 10px is much more reasonable without preventing zooms into pretty large areas.